app/models/service_orchestration.rb

```diff
@@ -192,6 +192,8 @@ def orchestration_runner_job
 
   def wait_on_orchestration_stack
     while deploy_stack_job.orchestration_stack.blank?
+      raise _("Orchestration template runner finished unexpectedly. Check evm.log for details.") if deploy_stack_job.dispatch_status == 'finished'
```

Comment:
is it possible that the stack finishes normally (for a simple quick deployment) before we enter into the wait_on mode?

Reply:
If the stack already exists, the job would wait on the stack until the job times out and the worker got killed.